modules/tinymce/src/plugins/link/main/ts/core/Actions.ts

```diff
@@ -39,11 +41,12 @@ const gotoSelectedLink = (editor: Editor) => (): void => {
 };
 
 const setupGotoLinks = (editor: Editor): void => {
-  editor.on('click', (e) => {
-    const link = getLink(editor, e.target);
-    if (link && VK.metaKeyPressed(e)) {
+  editor.on('click', (e: any) => {
+    const links = Utils.getLinks(editor.dom.getParents(e.target)) as [HTMLAnchorElement];
```

Comment:
I'm a little surprised you refactored the code above without using it here. Is there a test for the refactored code at least?

Reply:
It would require the event target `e.target` to be passed into the refactored code, which makes harder to understand so I decided not to use it.

I will add few more tests for the refactored code